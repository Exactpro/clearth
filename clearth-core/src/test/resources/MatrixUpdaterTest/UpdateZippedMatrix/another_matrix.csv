#Id,#GlobalStep,#Action
id1,Step1,SetStatic