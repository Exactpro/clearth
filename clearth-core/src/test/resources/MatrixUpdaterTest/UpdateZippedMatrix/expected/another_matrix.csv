#Id,#GlobalStep,#Action,#AnotherCharacter
id1,Step1,SetStatic,A
